qiskit_experiments/library/randomized_benchmarking/rb_utils.py

```diff
@@ -294,3 +310,264 @@ def calculate_2q_epg(
             out[qubit_pair] = {gate_2_qubit_type: 3 / 4 * (1 - alpha_c_2q) / n_gate_2q}
 
         return out
+
+
+class QubitGateTuple:
```

Comment:
This looks like an unnecessary class, you could just as easily use the tuple directly.